build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -98,7 +119,7 @@ jobs:
     displayName: 'Publish artifact: nupkg'
     inputs:
       PathtoPublish: '${{ parameters.nupkgdir }}'
-      artifactName: 'drop'
+      artifactName: 'nuget'
```

Comment:
I have a precise directory that I use to do the nuget signing, this will have to be adjusted/re-tested after you completee this PR.

Reply:
To be explicit, contact me and let's run the release pipeline together (I have to trigger it, I have the signing authorization).